workflow_calcium_imaging/ingest.py

```diff
@@ -17,8 +17,12 @@
 def ingest_subjects(
     subject_csv_path="./user_data/subjects.csv", skip_duplicates=True, verbose=True
```

Comment:
ideally, this PR would include typing suggestions for each argument
```suggestion
    subject_csv_path:str="./user_data/subjects.csv", skip_duplicates:bool=True, verbose:bool=True
```